packages/lesswrong/components/ea-forum/giving-portal/ElectionCandidatesList.tsx

```diff
@@ -80,7 +80,7 @@ const ElectionCandidatesList = ({type="preVote", selectedCandidateIds, onSelect,
   classes: ClassesType,
 }) => {
   const isSelect = type === "select";
-  const [sortBy, setSortBy] = useState<ElectionCandidatesSort | "random">("mostPreVoted");
+  const [sortBy, setSortBy] = useState<ElectionCandidatesSort | "random">(isSelect ? "random" : "mostPreVoted");
```

Comment:
Fix for a bug introduced by https://github.com/ForumMagnum/ForumMagnum/pull/8345 (mostPreVoted isn't an option if `type` is "select" so it throws an error if that is the default sorting). I'll also put this up as a separate PR